tests/cut-plane/test_cli.py

```diff
@@ -16,15 +16,15 @@ def test_cli_compute_file():
     runner = CliRunner()
 
     # vanilla test
-    result = runner.invoke(cli, ['compute', 'file', NEURON_SLICE])
+    result = runner.invoke(compute, ['file', NEURON_SLICE])
     if result.exit_code:
         raise AssertionError(result.exception)
 
     # test writing output
     with TemporaryDirectory(prefix='test_cli_compute_file') as temp_dir:
```

Comment:
As in NeuroR I think this could be replaced by the `tmpdir` fixture.

Reply:
thank you, done